diff --git a/videocodec/Android.mk b/videocodec/Android.mk
index 0676fcb..0220c72 100644
--- a/videocodec/Android.mk
+++ b/videocodec/Android.mk
@@ -42,7 +42,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderAVC.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderAVC
 ifeq ($(TARGET_BOARD_PLATFORM),clovertrail)
@@ -104,7 +104,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderVP8.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderVP8
 
@@ -176,7 +176,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderVP9HWR.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderVP9HWR
 
@@ -223,7 +223,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderVP9Hybrid.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderVP9Hybrid
 
@@ -275,7 +275,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderMPEG4.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderMPEG4
 ifeq ($(TARGET_BOARD_PLATFORM),clovertrail)
@@ -332,7 +332,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderH263.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderH263
 ifeq ($(TARGET_BOARD_PLATFORM),clovertrail)
@@ -389,7 +389,7 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderWMV.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderWMV
 ifeq ($(TARGET_BOARD_PLATFORM),clovertrail)
@@ -454,7 +454,7 @@
 LOCAL_SHARED_LIBRARIES += libdl
 endif
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderAVCSecure
 
@@ -495,7 +495,7 @@
     OMXVideoEncoderAVC.cpp
 
 LOCAL_CFLAGS += $(LOCAL_C_FLAGS)
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoEncoderAVC
 include $(BUILD_SHARED_LIBRARY)
@@ -537,7 +537,7 @@
 ifeq ($(SW_MPEG4_ENCODER),true)
     LOCAL_CFLAGS += -DSYNC_MODE
 endif
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoEncoderH263
 include $(BUILD_SHARED_LIBRARY)
@@ -576,7 +576,7 @@
 
 LOCAL_CFLAGS += $(LOCAL_C_FLAGS)
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoEncoderMPEG4
 include $(BUILD_SHARED_LIBRARY)
@@ -609,9 +609,10 @@
     OMXVideoDecoderBase.cpp \
     OMXVideoDecoderPAVC.cpp
 
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoDecoderPAVC
+
 ifeq ($(TARGET_BOARD_PLATFORM),clovertrail)
 LOCAL_CFLAGS += -DVED_TILING
 endif
@@ -659,7 +660,7 @@
     OMXVideoEncoderVP8.cpp
 
 LOCAL_CFLAGS += $(LOCAL_C_FLAGS)
-LOCAL_CFLAGS += -Werror
+#LOCAL_CFLAGS += -Werror
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libOMXVideoEncoderVP8
 include $(BUILD_SHARED_LIBRARY)
diff --git a/videocodec/libvpx_internal/libvpx.mk b/videocodec/libvpx_internal/libvpx.mk
index 5a95803..7df2470 100644
--- a/videocodec/libvpx_internal/libvpx.mk
+++ b/videocodec/libvpx_internal/libvpx.mk
@@ -123,7 +123,7 @@ LOCAL_GENERATED_SOURCES += $(VPX_GEN)
 
 ifeq ($(libvpx_target),x86)
 libvpx_x86_asm_src_files := $(filter %.asm, $(libvpx_codec_srcs_unique))
-libvpx_x86_yasm_dir := prebuilts/misc/$(HOST_OS)-$(HOST_PREBUILT_ARCH)/yasm
+libvpx_x86_yasm_dir := /usr/bin
 
 X86_ASM_GEN := $(addprefix $(libvpx_intermediates)/, $(libvpx_x86_asm_src_files:%.asm=%.asm.o))
 
